src/navigation/src/SimpleNavigator.cpp

```diff
@@ -12,17 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+// Navigation Strategy based on:
+// Brock, O. and Oussama K. (1999). High-Speed Navigation Using the Global Dynamic Window Approach. IEEE.
+// https://cs.stanford.edu/group/manips/publications/pdfs/Brock_1999_ICRA.pdf
+
 #include <string>
 #include <memory>
 #include <exception>
 #include <chrono>
+#include <iostream>
```

Comment:
I don't see that iostream is required. Still using it? Should remove any #includes that aren't required.

Reply:
Removed